### ISSUE TITLE
Spurious "more indices than dimensions" errors?

### ISSUE THREAD

**Issue by sbromberger** [tonyhffong/Lint.jl#75]
I created a (somewhat) minimal test case in `testlint.jl`:
```julia
function t1(n::Integer)
  parents = zeros(Int, (n,n))
  if (n > 3)
    parents[2,2] = 5
    parents[1,3] = 7
  end
  parents
end
```

`lintfile("testlint.jl")` produces
```
linttest.jl:4 [t1             ] ERROR  parents[2,2] has more indices than dimensions
linttest.jl:5 [t1             ] ERROR  parents[1,3] has more indices than dimensions
```

I don't think this is correct, as the function behaves as expected for a variety of inputs.

**Comment by tonyhffong** [comment]
It should be fixed now. Thanks for the report.

**Comment by sbromberger** [comment]
Thanks - confirmed that I'm not getting these errors any more with the latest master.
